src/components/table/OcTable.vue

```diff
@@ -164,6 +172,9 @@ export default {
     },
   },
   computed: {
+    tableData() {
+      return this.sortedData || this.data
```

Comment:
Looks to me like `this.sortedData` always returns, well, data. Why do we need the `|| this.data`? 🤔

Reply:
it's a fallback chain because if we have the custom sorters as discussed yesterday and one mixinData returns nothing it falls back to the default data.

```
this.sortedData || this.paginatedData || this.data

Reply:
sorted data will be paginated as well... the fallback feels weird to me

Reply:
please have a look at latest commit, i think it make now more sense

Reply:
👍